c_tests/tests/compile_simple.c

```diff
@@ -12,16 +12,18 @@
 #include <yk_testing.h>
 
 int main(int argc, char **argv) {
-  void *tt = __yktrace_start_tracing(HW_TRACING);
-  int res = 1 + 1;
+  int res = 0;
+  void *tt = __yktrace_start_tracing(HW_TRACING, &res);
+  res = 1 + 1;
```

Comment:
I suspect that even in `-O0` this will become `res = 2`.

Reply:
Probably. LLVM does mandatory const folding during ir generation, so if no alloca are used for the temp results, this will always be const folded.

Reply:
Yeah, it does, but that doesn't change the outcome of this test. But I can change it to `res = 2` if you like.

Reply:
We should change it because `1+1` is kinda misleading in the context of what we care about (we'll probably only ever see IR for `2`).

Reply:
Fixed in 7731d53.